shell.nix

```diff
@@ -13,6 +13,7 @@ stdenv.mkDerivation {
         customMemoryManagement = false;
       })
       autoreconfHook
+      perlPackages.DBDSQLite
```

Comment:
See the PR description; it's not at all related to the doc change, but I think it should be that way anyway? I can split it into another PR if you want.